examples/durable/DurableApp/HumanInteractionOrchestrator/run.ps1

```diff
@@ -0,0 +1,28 @@
+using namespace System.Net
+
+param($Context)
+
+Write-Host 'HumanInteractionOrchestrator: started.'
+
+$output = @()
+
+$duration = $Context.Input.Duration
+
+Invoke-ActivityFunction -FunctionName "RequestApproval"
```

Comment:
This is supposed to be (eventually) working code. As opposed to samples in the docs that don't have to be directly runnable, this code should work as is, so we need to add at least very simple implementations of RequestApproval, ProcessApproval, and EscalateApproval functions. Very simple implementations that just log a message will do.